module_utils/network_lsr/argument_validator.py

```diff
@@ -521,6 +521,28 @@ def _validate_post(self, value, name, result):
         return result
 
 
+class ArgValidator_DictWireless(ArgValidatorDict):
+
+    VALID_KEY_MGMT = [
+        "wpa-psk",
+        "wpa-eap",
+    ]
+
+    def __init__(self):
+        ArgValidatorDict.__init__(
+            self,
+            name="wireless",
+            nested=[
+                ArgValidatorStr("ssid"),
+                ArgValidatorStr(
+                    "key-mgmt", enum_values=ArgValidator_DictWireless.VALID_KEY_MGMT
+                ),
+                ArgValidatorStr("password"),
```

Comment:
This has a minimum and maximum length, too, IIRC.

Reply:
From what I looked up, the password should be less than 63 bytes. Have added a check for this.